mptt/models.py

```diff
@@ -341,6 +342,17 @@ def register(meta, cls, **kwargs):
                     if field_name not in existing_field_names:
                         field = models.PositiveIntegerField(db_index=True, editable=False)
                         field.contribute_to_class(cls, field_name)
+                        
+                if django.VERSION >= (1, 11):
+                    
+                    tree_id_attr = getattr(cls._mptt_meta, 'tree_id_attr')
```

Comment:
I don't think you need `getattr()` here. `cls._mptt_meta.tree_id_attr` should do.